modules/weather.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+let cache = require('./cache.js');
+
+module.exports = getWeather;
+
+function getWeather(latitude, longitude) {
+  const key = 'weather-' + latitude + longitude;
+  const url = `http://api.weatherbit.io/v2.0/forecast/daily/?key=${WEATHER_API_KEY}&lang=en&lat=${lat}&lon=${lon}&days=5`;
```

Comment:
- During demo today, Ryan showed us where some errors were with this. 
- The function takes in `latitude` and `longitude` but on line we are passing in `${lat}&lon=${lon}`
- They should be `latitude` and `longitude` 👍